operator/src/test/java/org/keycloak/operator/RealmImportE2EIT.java

```diff
@@ -0,0 +1,169 @@
+                                .withImage("curlimages/curl:7.78.0")
+                                .withRestartPolicy("Never")
+                                .build())
+                        .done();
+                Log.info("Waiting for curl Pod to finish running");
+                Awaitility.await().atMost(2, MINUTES)
+                        .until(() -> {
+                            String phase =
+                                    k8sclient.pods().inNamespace(namespace).withName("curl").get()
+                                            .getStatus().getPhase();
+                            return phase.equals("Succeeded") || phase.equals("Failed");
+                        });
+
+                String curlOutput =
+                        k8sclient.pods().inNamespace(namespace)
+                                .withName(curlPod.getMetadata().getName()).getLog();
+                Log.info("Output from curl: '" + curlOutput + "'");
+                assertThat(curlOutput).isEqualTo("200");
+            } catch (KubernetesClientException ex) {
+                throw new AssertionError(ex);
+            } finally {
+                Log.info("Deleting curl Pod");
+                k8sclient.pods().inNamespace(namespace).withName("curl").delete();
+                Awaitility.await().atMost(1, MINUTES)
+                        .until(() -> k8sclient.pods().inNamespace(namespace).withName("curl")
+                                .get() == null);
+            }
+        });
+    }
+
+    @Test
+    public void testNotWorkingRealmImport() {
+        Log.info(((operatorDeployment == OperatorDeployment.remote) ? "Remote " : "Local ") + "Run Test :" + namespace);
+        // Arrange
+        k8sclient.load(getClass().getResourceAsStream("/example-postgres.yaml")).inNamespace(namespace).createOrReplace();
+        k8sclient.load(getClass().getResourceAsStream("/example-keycloak.yml")).inNamespace(namespace).createOrReplace();
+
+        // Act
+        k8sclient.load(getClass().getResourceAsStream("/incorrect-realm.yaml")).inNamespace(namespace).createOrReplace();
+
+        // Assert
+        Awaitility.await()
+                .atMost(3, MINUTES)
+                .pollDelay(5, SECONDS)
+                .ignoreExceptions()
+                .untilAsserted(() -> {
+                    var conditions = k8sclient
+                            .resources(KeycloakRealmImport.class)
+                            .inNamespace(namespace)
+                            .withName("example-count0-kc")
+                            .get()
+                            .getStatus()
+                            .getConditions();
+
+                    assertThat(getCondition(conditions, HAS_ERRORS).getStatus()).isTrue();
+                    assertThat(getCondition(conditions, DONE).getStatus()).isFalse();
+                    assertThat(getCondition(conditions, STARTED).getStatus()).isFalse();
+                });
+    }
```

Comment:
I would have a test involving a realmimport using a name of an already existing realm.

Reply:
it will simply succeed as it works today.

Reply:
In fact it works....at least locally , with Minikube.
Tunnels are only needed for LoadBalancers :
https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-tunnel

$ kubectl run --image=vad1mo/hello-world-rest --port=5050 hello2 -n
yournamespace
$ kubectl apply -f myservice.yaml -n yournamespace

myservice.yaml ---------------------------
kind: Service
apiVersion: v1
metadata:
  name: my-service
spec:
  ports:
    - protocol: TCP
      port: 5050
      targetPort: 5050
      nodePort: 30007
  selector:
    run: hello3
  type: NodePort

❯ curl $(minikube ip):30007
/ - Hello World! Host:hello3/172.17.0.7%

❯ minikube profile list
|----------|-----------|---------|--------------|------|---------|---------|-------|
| Profile  | VM Driver | Runtime |      IP      | Port | Version | Status
 | Nodes |
|----------|-----------|---------|--------------|------|---------|---------|-------|
| minikube | docker    | docker  | 192.168.58.2 | 8443 | v1.22.3 | Running
|     1 |
|----------|-----------|---------|--------------|------|---------|---------|-------|